owlbot.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Google LLC
```

Comment:
Huh I wonder why git didn't think this was a file rename like in other PRs.

Reply:
Let me retry the migration and see if that fixes it.

Reply:
I tried renaming the file using `git mv` but the `synth.py` file still shows as deleted and `owlbot.py` added in github. My guess is that when the changes are significant github can't display the renaming properly.